CONTRIBUTING.rst

```diff
@@ -0,0 +1,153 @@
+How to contribute to Flask-SQLAlchemy
+=====================================
+
+Thank you for considering contributing to Flask-SQLAlchemy!
+
+Support questions
+-----------------
+
+Please, don't use the issue tracker for this. Use one of the following
+resources for questions about your own code:
+
+* The IRC channel ``#pocoo`` on FreeNode.
+* The IRC channel ``#python`` on FreeNode for more general questions.
+* The mailing list flask@python.org for long term discussion or larger issues.
+* Ask on `Stack Overflow`_. Search with Google first using:
+  ``site:stackoverflow.com flask-sqlalchemy {search term, exception message, etc.}``
+
+.. _Stack Overflow: https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked
+
+Reporting issues
+----------------
+
+- Describe what you expected to happen.
+- If possible, include a `minimal, complete, and verifiable example`_ to help
+  us identify the issue. This also helps check that the issue is not with your
+  own code.
+- Describe what actually happened. Include the full traceback if there was an
+  exception.
+- List your Python, Flask, and Werkzeug versions. If possible, check if this
+  issue is already fixed in the repository.
+
+.. _minimal, complete, and verifiable example: https://stackoverflow.com/help/mcve
+
+Submitting patches
+------------------
+
+- Include tests if your patch is supposed to solve a bug, and explain
+  clearly under which circumstances the bug happens. Make sure the test fails
+  without your patch.
+- Try to follow `PEP8`_, but you may ignore the line length limit if following
+  it would make the code uglier.
+
+First time setup
+~~~~~~~~~~~~~~~~
+
+- Download and install the `latest version of git`_.
+- Configure git with your `username`_ and `email`_::
+
+        git config --global user.name 'your name'
+        git config --global user.email 'your email'
+
+- Make sure you have a `GitHub account`_.
+- Fork Flask-SQLAlchemy to your GitHub account by clicking the `Fork`_ button.
+- `Clone`_ your GitHub fork locally::
+
+        git clone https://github.com/{username}/flask-sqlalchemy
+        cd flask-sqlalchemy
+
+- Add the main repository as a remote to update later::
+
+        git remote add pallets https://github.com/pallets/flask-sqlalchemy
+        git fetch pallets
+
+- Create a virtualenv::
+
+        python3 -m venv env
+        . env/bin/activate
+        # or "env\Scripts\activate" on Windows
+
+- Install Flask-SQLAlchemy in editable mode with development dependencies::
+
+        pip install -e ".[dev]"
```

Comment:
We don't have dev dependencies specified yet, but I think it's a good idea to make that happen.

Reply:
I put in and then closed this issue https://github.com/pallets/flask-sqlalchemy/issues/725 to update `setup.py` after speaking with @davidism. Should this instead say ` pip install -e .`? I

Reply:
I went ahead and updated it to `pip install -e .`